### PR TITLE
Issue #3:  improved diff on Export / Dump.

### PR DESCRIPTION
**Pull request by kodecharlie**
Per issue, this improvement addresses re-arrangements of original tree elements.
However, per discussions, it still has shortcomings:  adding a new group/assignment/etc; moving between groups, and so on.

### CODE REVIEW

File: src/com/cdd/bao/editor/CompareVocabTree.java [hunk 1]
```diff
@@ -28,6 +28,8 @@
 import java.io.*;
 import java.util.*;
 
+import org.apache.commons.lang3.StringUtils;
```
Comment (aclarkxyz) [inline src/com/cdd/bao/editor/CompareVocabTree.java]: Use wildcards.

File: src/com/cdd/bao/editor/CompareVocabTree.java [hunk 2]
```diff
@@ -135,25 +137,81 @@ public void show()
 	}
 	
 	// ------------ private methods ------------
+	
+	private static final class OldNewPair
+	{
+		SchemaVocab.StoredTree tree1;	// old
```
Comment (aclarkxyz) [inline src/com/cdd/bao/editor/CompareVocabTree.java]: Single space before comment.

**Comment by aclarkxyz** [comment]
First test: loaded up schema.ttl; swap order of first two assignments (bioassay <-> bioassay type); File|Export to vocab.dump ... it claims that these first 2 assignments have completely different values. It should report that nothing interesting has happened (at least for this most simple of cases).

**Comment by kodecharlie** [comment]
@aclarkxyz:  it's a boundary case (in this case, the "top" of the edit-tree panel).  I'm looking into it.

**Comment by kodecharlie** [comment]
Per morning discussion, I have nixed "issue-#3" branch and merged changes from this branch into the "json-serial" branch (issue-#48).

**Pull request closed without merging**